tools/Resources.tt

```diff
@@ -0,0 +1,226 @@
+<#@ template hostspecific="true" #>
```

Comment:
What's this for?

Reply:
It's a custom generator for Resources.Designer.cs files inherited from `dotnet-ef`. I removed it completely in #8405. (As I said in the description, this is just temporary.)